### ISSUE TITLE
Stuck at splash screen

### ISSUE THREAD

**Issue by b0wter** [AmProsius/gothic-1-community-patch#398]
### Please agree to the following

- [X] I have searched [existing issues](https://github.com/AmProsius/gothic-1-community-patch/issues?q=) for duplicates

### Describe the bug

I am running Windows 11. After installing the GOG version of Gothic 1 I've installed all perquisites (see steps to reproduce and the following image):
![image](https://github.com/user-attachments/assets/ae2fa1d5-71db-4a84-8594-6db5a6776de7)

Whenever I try to start the game the launcher starts fine but I am stuck at the Gothic splash screen and have to use the task manager to kill the process.

### Expected behavior

The game should launch

### Steps to reproduce the issue

1. install GOG Galaxy
2. install Gothic 1
3. install Playerkit 1.0k
4. install ninja 3.1.15
5. install G1CP-1.2.0.exe
6. start game
7. click "Play the game" in Gothic Launcher
8. get stuck at Gothic splash screen

### Additional context

I've installed the game through GOG. I run that latest version of Windows 11

**Comment by szapp** [comment]
Hi @b0wter 

Have you installed 1.08k_mod? See here for details:

https://github.com/AmProsius/gothic-1-community-patch#requirements

**Comment by b0wter** [comment]
Thanks for the fast reply!
I meant that patch when I was referring to „Playerkit 1.0k“. Must have missed a letter there.

**Comment by szapp** [comment]
Can you provide a link to that patch? The names of different patches are very similar, e.g. Playerkit 1.0k ≠ Version 1.08k_mod

**Comment by b0wter** [comment]
I was following the instructions posted here:

https://www.gog.com/forum/gothic_series/introducing_gothic_1_community_patch

The post says "Playerkit 1.0k“, but is actually linking to 1.08k_mod:

https://www.worldofgothic.de/dl/download_34.htm

**Comment by szapp** [comment]
Go through this checklist: 
1. Gothic must not be installed in _C:\Program Files_ or _C:\Program Files (x86)_.
2. Gothic should be started with administrative privileges. 
3. Data Execution Prevention (DEP) is either disabled or the Gothic executables are added as exceptions. See the [Spine FAQ No. 8](https://clockwork-origins.com/spine/#faq-question-1576926774006) for reference (these instructions hold even when _not_ using Spine).
4. It's advisable to easier use SystemPack or Union to make Gothic run stable on newer machines.

**Comment by b0wter** [comment]
Thanks for all the help. I finally got it working after some testing. I am leaving a detailed description in case anyone encounters the same issue.
I‘ve uninstalled everything using the Add/Remove Program feature from Windows.
Then I tested the game after each step while reinstalling:

_I installed the base game using GOG Galaxy_
=> game starts und runs

_I installed „Playerkit 1.08k mod“_
=> game gets stuck at splash screen

_I used spine to start the game (using the system pack)_
=> game crashes with access violation

_I configured DEP according to the spine documentation_
=> game still crashes with access violaten

_I Installed ninja_
=> game starts in a weird state with an empty console window and a window that only consists of the titlebar and quits after a few seconds

_I installed the Community Patch_
=> same result as above

_I installed union using spine_
=> game now runs properly

_I used spine to configure Gothic properly_
